lib/elixir/test/elixir/exception_test.exs

```diff
@@ -378,6 +378,16 @@ defmodule ExceptionTest do
            """
   end
 
+  test "UndefinedFunctionError when the mfa is a macro but require wasn't called" do
+    assert %UndefinedFunctionError{module: Integer, function: :is_odd, arity: 1} |> message == """
+           function Integer.is_odd/1 is undefined but Integer defines a macro with the same name and arity. Maybe you meant to call:
```

Comment:
-"call"? Seems clearer since some people might not know that require is actually a macro itself, and thus is being called.